File: app/src/main/java/rus/tutby/utils/MyDisplayMetrics.java
package rus.tutby.utils;


import android.util.DisplayMetrics;
import android.util.Log;
import android.view.WindowManager;

public class MyDisplayMetrics {

    public static float getWidth(WindowManager windowManager) {
        DisplayMetrics displayMetrics = new DisplayMetrics();
        windowManager.getDefaultDisplay().getMetrics(displayMetrics);
        return displayMetrics.widthPixels / displayMetrics.density;
    }

    public static float getHeight(WindowManager windowManager) {
        DisplayMetrics displayMetrics = new DisplayMetrics();
        windowManager.getDefaultDisplay().getMetrics(displayMetrics);
        return displayMetrics.heightPixels / displayMetrics.density;
    }

    public static int getWidthInPixels(WindowManager windowManager) {
        DisplayMetrics displaymetrics = new DisplayMetrics();
        windowManager.getDefaultDisplay().getMetrics(displaymetrics);
        return displaymetrics.widthPixels;
    }

    public static int getHeightInPixels(WindowManager windowManager) {
        DisplayMetrics displaymetrics = new DisplayMetrics();
        windowManager.getDefaultDisplay().getMetrics(displaymetrics);
        return displaymetrics.heightPixels;
    }
}

File: app/src/main/java/rus/tutby/utils/MyDisplayMetrics.kt
package rus.tutby.utils;


import android.util.DisplayMetrics;
import android.util.Log;
import android.view.WindowManager;

class MyDisplayMetrics() {

    companion object {
        fun getWidth(windowManager: WindowManager): Float {
            val displayMetrics = DisplayMetrics();
            windowManager.defaultDisplay.getMetrics(displayMetrics);
            return displayMetrics.widthPixels / displayMetrics.density;
        }

        fun getHeight(windowManager: WindowManager): Float {
            val displayMetrics = DisplayMetrics();
            windowManager.defaultDisplay.getMetrics(displayMetrics);
            return displayMetrics.heightPixels / displayMetrics.density;
        }

        fun getWidthInPixels(windowManager: WindowManager): Int {
            val displayMetrics = DisplayMetrics();
            windowManager.defaultDisplay.getMetrics(displayMetrics);
            return displayMetrics.widthPixels;
        }

        fun getHeightInPixels(windowManager: WindowManager): Int {
            val displayMetrics = DisplayMetrics();
            windowManager.defaultDisplay.getMetrics(displayMetrics);
            return displayMetrics.heightPixels;
        }
    }
    
}